Quality of life improvements to system tests

Karl-G1 opened the pull request:
- [x] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/niveristand-aim-arinc429-custom-device/blob/main/CONTRIBUTING.md).

### What does this Pull Request accomplish?

- Deploy once during Loopback test class execution
  - This is the largest change and required resetting the Acyclic Tx channel values to work with one deployment
  - Use new testing VI calls in globalSetUp and setUp VIs to retain class data 
- Delete scripted system definition files after tests execute to clean up generated assets
- Fix bug in `Generate VS Channels Info.vi` connector pane / VI icon
- Use an enum as a required input for selecting label type in `Generate VS Channels Info.vi`
  - Update all sub VIs that call this VI to change the input to the enum
- Use a config file to specify PXI slot number

### Why should this Pull Request be merged?

This PR is a sizable collection of quality of life improvements for this repo's system tests. 
The biggest visible changes are 1) shorter test times and 2) not leaving scripted assets on disk after test runs.

### What testing has been done?
Ran the automated tests to see a ~70 second time reduction by not deploying multiple times for the main loopback tests.

<details>
  <summary>VI Tester results after changes</summary>

  ![image](https://user-images.githubusercontent.com/31290917/182706133-19a3c950-a6e3-4c00-bc4e-bc245c654649.png)

</details>

<details>
  <summary>VI Tester results before changes</summary>

  ![image](https://user-images.githubusercontent.com/31290917/182707304-509bae55-033e-4bef-a198-473babf289cb.png)

</details>

buckd reviewed (approved):
- [ ] Remove the breakpoint in `test Acyclic Loopback Selective Triggers.vi`.

Karl-G1 commented:
> * [ ]  Remove the breakpoint in `test Acyclic Loopback Selective Triggers.vi`.

When I went to remove the breakpoint in the VI, I found that I could not open any of the modified test VIs due to a LabVIEW crash (Fatal Internal Error 0xE59037CA: ...). This reproduced on another dev machine as well, so I checked out the files from main, updated the string constant to an enum, and added the sub VI in the acyclic test VIs.
